src/components/category/Category.jsx

```diff
@@ -0,0 +1,40 @@
+
+import axios from 'axios';
+import React ,{useState,useEffect} from 'react'
+import { Link } from 'react-router-dom';
+
+import './Category.css';
+
+function Category(){
+
+const [category , setCategory ] = useState([]);
+
+ useEffect(()=>{
+
+    (async()=>{
+
+        try {
+            const response = await axios.get("/api/categories");
```

Comment:
const {data} = await axios...

Reply:
yeah , thanks for suggestion i will use this